SasquatchTV/SasquatchTV/ViewControllers/AnalyticsViewController.swift

```diff
@@ -11,6 +11,7 @@ enum AnalyticsActionsRows : Int {
 
 class AnalyticsViewController : UIViewController, UITableViewDataSource, AppCenterProtocol {
 
+  @IBOutlet weak var enableManualSessionTracker : UISegmentedControl?;
```

Comment:
Why this ends with a question mark. I think it should be `!` for all outlet. Please test it and change all to `!` if it works.